### ISSUE TITLE
Ambiguïté entre le glossaire RAWeb et les exemples du WCAG 2.1

### ISSUE THREAD

**Issue by suisoku** [accessibility-luxembourg/ReferentielAccessibiliteWeb#19]
Hello,

Dans le [Glossaire #U](https://accessibilite.public.lu/fr/raweb1/glossaire.html#uniquement-a-des-fins-de-presentation) il est mentionné ceci :

```md
Note 1 : l’utilisation d’éléments <div> ou <span> ou plusieurs <br> pour créer visuellement 
un paragraphe est considérée comme non conforme et invalide le critère.

Exemple : <div>, paragraphes d’un bloc de texte simulés, <br>, à l’aide de plusieurs balises <br>, </div>
````

Mais cela reste ambigu pour les éléments qui ont plusieurs fonctions sémantiques, par exemple :

Du texte dans une balise `span` qui peut être considéré visuellement comme un paragraphe, mais qui est en réalité un "label", car cette balise est la cible d'un `aria-labelledby`.

Dans le [WCAG - ARIA16](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA16) plusieurs exemples avec ou sans `p` (Exemple 3), sont présents sur les textes `aria-labelledby`. Dans ce cas précis la conformité ou non-conformité reste ambiguë dans le RAWeb.

D'autres exemples de textes sans `<p>` comme le `aria-describedby` sont également présents dans le WCAG : [Using the aria-describedby property to provide a descriptive label](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA1).

Je propose qu'on explicite davantage les cas où l'utilisation du texte sans balise sémantique est non-conforme dans le glossaire.

**Comment by AlainVagner** [comment]
Bonjour, je suppose que vous parlez du critère 8.9 qui est lié à l'entrée de glossaire "uniquement à des fins de présentation".

Le fait qu'un `span` ait un attribut `aria-labelledby` n'en fait pas un élément `label`. Il faut d'ailleurs se méfier des attributs `aria-labelledby` et `aria-describedby` sur les éléments non interactifs car le support des technologies d'assistance est relativement limité.
La technique `ARIA16` s'applique bien à des éléments interactifs, ce qui n'est pas l'objet du critère 8.9, elle ne fait pas partie des correspondances du 8.9. Considérez plutôt à la place les techniques qui y sont mentionnées : [F43](https://www.w3.org/WAI/WCAG21/Techniques/failures/F43), [F92](https://www.w3.org/WAI/WCAG21/Techniques/failures/F92), [G115](https://www.w3.org/WAI/WCAG21/Techniques/general/G115), [H88](https://www.w3.org/WAI/WCAG21/Techniques/html/H88).

**Comment by suisoku** [comment]
Bonjour,  merci de votre réponse rapide : ) 

En effet, c'est dans le cadre du critère 8.9 du RAWEB que nous nous posant la question au sein de notre pôle accessibilité. Nous essayons de détecter toutes les **non-conformités** liés au critère 8.9 sur les applications du groupe, notamment les textes bruts qui ne sont pas dans des balises sémantiques (`<li>`, `<p>` etc.). 

Cependant la question s'est posé lorsqu'on effectue un audit sur nos composants de saisie : Est-ce que du point de vue du critère 8.9, du texte / une phrase écrite directement dans un `<span> / <div>`  et qui "peut" s'apparenter visuellement à un paragraphe est toujours **conforme** vis-à-vis  de la note du glossaire mentionné plus haut? Certains de nos composants sont constitués d'un `<span id=..>` qui sert d'étiquette visible  et d'un `<input aria-labelledby=... >`

**Comment by AlainVagner** [comment]
Si vous avez du texte qui s'apparente à un paragraphe il doit être dans une balise `<p>`. Je ne pense pas que vous ayez un paragraphe de texte comme label d'un champ de formulaire.

**Comment by suisoku** [comment]
En effet c'est plus clair maintenant, merci !
